packages/rancher-logging/overlay/templates/loggings/gke/logging.yaml

```diff
@@ -15,7 +15,10 @@ spec:
     inputTail:
       Tag: "gke"
       Path: "/var/log/kube-proxy.log"
-  {{- $total_tolerations := concat (.Values.tolerations) (.Values.fluentbit_tolerations) }}
+  {{- if .Values.fluentbit.mountPath }}
+    mountPath: {{ .Values.fluentbit.mountPath }}
```

Comment:
Curious about the validity of doing this on hosted providers like GKE and EKS, can you actually get to those logs or to that dir ? This ties in with the hosted provider logs issue I believe.

Reply:
If the directory is readable, then you should be able to. However, I think you pointed out that AKS/EKS/GKE should probably not have the mountPath exposed. I'll remove it for those

Reply:
I'm not actually sure either way, thought you might due to that provider logs issue. I would assume not though.

Reply:
Yep I'll remove it for the providers for now then.